docs/notebooks/08Calibrate_double_ended.ipynb

```diff
@@ -41,17 +41,18 @@
     "import warnings\n",
     "\n",
     "from dtscalibration import read_silixa_files\n",
-    "from dtscalibration.datastore_utils import (\n",
+    "from dtscalibration.dts_accessor_utils import (\n",
```

Comment:
I would prefer to put these under `dtscalibration` itself. I.e. 

```py
import dtscalibration
dtscalibration.shift_double_ended(ds, ...)
```
This also shouldn't trigger any circular imports as (this part of) the code is functional instead of object oriented.

Most users will use these functions, so it's best to make them easily accessible. Ideally end users don't need to know jargon like "accessor" at all.

Reply:
I now see that they're actually already in `dtscalibration/__init__.py`... So the docs can simply just be adjusted for this.